src/components/TextInput/BaseTextInput.js

```diff
@@ -253,7 +252,6 @@ class BaseTextInput extends Component {
                                         }}
                                         // eslint-disable-next-line
                                         {...inputProps}
-                                        defaultValue={this.state.value}
```

Comment:
> @eVoloshchak there are other components like ReportActionMessageEdit, RoomNameInput, StatePicker, EmojiPicker etc
> Will it affect them too?

I tried to check all of these components, everything seems to be working properly.
This happens thanks to this line in [BaseTextInput.js](https://github.com/Expensify/App/blob/main/src/components/TextInput/BaseTextInput.js)

https://github.com/Expensify/App/blob/28778a093d00dc8b882592886145e66d242d9159/src/components/TextInput/BaseTextInput.js#L23

So no matter if we pass `value` or `defaultValue`, it still sets it as `value` to state, and then it is passed to `RNTextInput`

Reply:
> Isn't it better to pass it as value for ReportActionMessageEdit and DatePicker?
> 
> Because it depends on the state, and it changes.
> 
> DefaultValue should be used only as the initial value, right?

Makes sense, will update shortly

Reply:
> Isn't it better to pass it as value for ReportActionMessageEdit and DatePicker?
> Because it depends on the state, and it changes.
> DefaultValue should be used only as the initial value, right?

That's right. I've updated the `ReportActionMessageEdit`, but not sure about the `DatePicker`
It seems like it is [using](https://github.com/Expensify/App/blob/28778a093d00dc8b882592886145e66d242d9159/src/components/DatePicker/index.js#L23) DefaultValue as the initial value
```javascript
  /* We're using uncontrolled input otherwise it wont be possible to
  * raise change events with a date value - each change will produce a date
  * and make us reset the text input */
  this.defaultValue = props.defaultValue
      ? moment(props.defaultValue).format(CONST.DATE.MOMENT_FORMAT_STRING)
      : '';
```